conf/routes

```diff
@@ -40,6 +40,7 @@ PUT /api/campaigns/:id/driver/:driverId     controllers.CampaignApi.acceptSugges
 PUT /api/campaigns/:id/not-driver/:driverId controllers.CampaignApi.rejectSuggestedCampaignTrafficDriver(id: String, driverId: Long)
 GET /api/campaigns/:id/driverstats          controllers.CampaignApi.getCampaignTrafficDriverStats(id: String)
 GET /api/campaigns/:id/ctastats             controllers.CampaignApi.getCampaignCtaStats(id: String)
+GET /api/campaigns/:id/referrals            controllers.CampaignApi.getCampaignReferrals(id: String)
```

Comment:
I've done `/api/v2` for now just so we mark the new endpoints. Then when we eventually clear all the old ones out we can remove the version extension.